static/app/components/notificationActions/notificationActionManager.tsx

```diff
@@ -0,0 +1,219 @@
+  const pagerdutyIntegrations: Record<number, AvailableNotificationAction[]> = {};
+  availableServices[NotificationActionService.PAGERDUTY].forEach(service => {
+    const integrationId = service.action.integrationId;
+    if (integrationId) {
+      if (integrationId in pagerdutyIntegrations) {
+        pagerdutyIntegrations[integrationId].push(service);
+      } else {
+        pagerdutyIntegrations[integrationId] = [service];
+      }
+    }
+  });
+
+  function renderNotificationActions() {
+    if (!notificationActions) {
+      return null;
+    }
+
+    const serviceActions: NotificationAction[] = [];
+
+    // Renders the notif actions grouped together by kind
+    Object.keys(actionsMap).forEach(serviceType => {
+      const services = actionsMap[serviceType];
+      serviceActions.push(
+        services.map(({action, index}) => (
+          <NotificationActionItem
+            key={index}
+            index={index}
+            defaultEdit={!action.id}
+            action={action}
+            recipientRoles={recipientRoles}
+            availableActions={availableServices[serviceType]}
+            pagerdutyIntegrations={pagerdutyIntegrations}
+            project={project}
+            onDelete={removeNotificationAction}
+            onUpdate={updateNotificationAction}
+          />
+        ))
+      );
+    });
+    return serviceActions;
+  }
+
+  function addNotificationAction(actionInstance: AvailableNotificationAction['action']) {
+    const updatedActions = cloneDeep(notificationActions);
+    updatedActions.push(actionInstance);
+    setNotificationActions(updatedActions);
+    updateAlertCount(parseInt(project.id, 10), updatedActions.length);
+  }
+
+  function removeNotificationAction(index: number) {
+    // Removes notif action from state using the index
+    const updatedActions = cloneDeep(notificationActions);
+    updatedActions.splice(index, 1);
+    setNotificationActions(updatedActions);
+    updateAlertCount(parseInt(project.id, 10), updatedActions.length);
+  }
+
+  function updateNotificationAction(index: number, updatedAction: NotificationAction) {
+    // Updates notif action from state using the index
+    const updatedActions = cloneDeep(notificationActions);
+    updatedActions.splice(index, 1, updatedAction);
+    setNotificationActions(updatedActions);
+  }
+
+  // The dropdown items for "Add Alert"
+  const getMenuItems = (): MenuItemProps[] => {
+    const menuItems: MenuItemProps[] = [];
+    Object.entries(availableServices).forEach(entry => {
+      const [serviceType, validActions] = entry;
+      if (validActions.length > 0) {
+        // Cannot have more than one Sentry notification
+        if (
+          serviceType === NotificationActionService.SENTRY_NOTIFICATION &&
+          actionsMap[serviceType].length === 1
+        ) {
+          return;
+        }
+
+        menuItems.push({
+          key: serviceType,
+          label: tct('Send a [service] notification', {
```

Comment:
using t here will return a string instead of a element
```suggestion
          label: t('Send a %s notification', 
```